### PR TITLE
Create svx-1.2.4.md

### PR DESCRIPTION
**Pull request by anastasiaUsik**

### CODE REVIEW

File: releases/svx-1.2.4.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# SVX 1.2.4 Release Notes
+
+**Software Release Date**: Nov 7, 2023
+
+**Summary**:
+
+The following is a summary of the key features and enhancements:
+
+* **Enhancements**: Updated context switcher options for administrators depending on their administrator levels.
```
Comment (javereec) [inline releases/svx-1.2.4.md]: @anastasiaUsik I don't understand what the updated context switcher does. Can you explain

**Review by javereec — CHANGES_REQUESTED** [review]
Should explain what this improved context switcher does

File: releases/svx-1.2.4.md (hunk 1 above)
Comment (anastasiaUsik) [reply releases/svx-1.2.4.md]: @javereec 
re-phrased proposal:
Improved the context-switching menu options based on the administrator levels

Explanation: If I'm only OA - I don't see a context switching menu, but when I've been added as a TA, this menu should appear for me with the option "return to Tenant dashboard", and etc

Comment (javereec) [reply releases/svx-1.2.4.md]: Ok, now I understand

**Pull request merged**